pocs/utils/images/__init__.py

```diff
@@ -224,29 +232,48 @@ def create_timelapse(directory, fn_out=None, file_type='jpg', **kwargs):
         field_name = head.split('/')[-2]
         cam_name = head.split('/')[-1]
         fname = '{}_{}_{}.mp4'.format(field_name, cam_name, tail)
-        fn_out = os.path.join(os.getenv('PANDIR'), 'images', 'timelapse', fname)
+        fn_out = os.path.normpath(os.path.join(directory, fname))
 
-    fn_dir = os.path.dirname(fn_out)
-    os.makedirs(fn_dir, exist_ok=True)
+    if verbose:
+        print("Timelapse file: {}".format(fn_out))
+
+    if os.path.exists(fn_out) and not overwrite:
+        raise FileExistsError("Timelapse exists. Set overwrite=True if needed")
+
+    ffmpeg = shutil.which('ffmpeg')
     inputs_glob = os.path.join(directory, '*.{}'.format(file_type))
```

Comment:
What is it that orders the files added to the video? Just the lexical order?

Reply:
I think ffmpeg is intelligent about ordering them but worth checking on. I usually sort after the glob but here we're just passing the glob.